ingestion/src/metadata/ingestion/source/dashboard/tableau.py

```diff
@@ -59,6 +60,17 @@
 
 
 class TableauSource(DashboardServiceSource):
+    """Tableau source entity class
+
+    Args:
+        config:
+        metadata_config:
+
+    Attributes:
+        config:
+        metadata_config:
+        all_dashboard_details:
+    """
```

Comment:
```suggestion
```
Do empty docstrings give any value? I think we can remove them or fill them with meaningful information, wdyt?

Reply:
Right I think we can remove them from here